src/core/Akka.Remote/Akka.Remote.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
   <ItemGroup>
     <EmbeddedResource Include="Configuration\Remote.conf" />
+    <ProjectReference Include="..\Akka.Streams\Akka.Streams.csproj" />
```

Comment:
Akka.Remote.Artery is coupled to Akka.Streams

Reply:
Makes sense